test/fixtures/transpile-css-prop/output.js

```diff
@@ -293,6 +339,12 @@ var _StyledP11 = (0, _styledComponents.default)("p")(_templateObject15(), functi
   return props.theme.color;
 });
 
-var _StyledButtonGhost = (0, _styledComponents.default)(Button.Ghost)(_templateObject16());
+var _StyledButtonGhost = (0, _styledComponents.default)(undefined)(_templateObject16());
+
+var _StyledButtonGhostNew = (0, _styledComponents.default)(undefined)(_templateObject17());
+
+var _StyledBox = (0, _styledComponents.default)(undefined)(_templateObject18());
```

Comment:
I'm pretty sure this will throw an error? I don't think we support this dynamic-as-prop-only usage.

Reply:
@mxstbr yeah, you're right. what do you suggest? maybe some random string? it shouldn't really matter since the component to render will always be passed in the `as` prop.